error/src/lib.rs

```diff
@@ -443,6 +443,10 @@ impl CaliptraError {
     pub const RUNTIME_CMD_RESERVED_PAUSER: CaliptraError = CaliptraError::new_const(0x000E004F);
     pub const RUNTIME_AUTH_AND_STASH_MEASUREMENT_DPE_ERROR: CaliptraError =
         CaliptraError::new_const(0x000E0050);
+    pub const RUNTIME_GET_IDEV_ID_UNPROVISIONED: CaliptraError =
+        CaliptraError::new_const(0x000E0051);
+    pub const RUNTIME_GET_IDEV_ID_UNSUPPORTED_ROM: CaliptraError =
```

Comment:
Maybe no action required, but can you double check that there's no overlap with the error codes in the `main-2.x` branch? Sam comment for all the errors in this file.

Ultimately I would like to pull the errors crate into a separate repo, but for now we may need to manually keep them in sync.

Reply:
These error messages are not yet taken in `main-2.x`, so it should be okay assuming there is no open PR using them.